{{cookiecutter.repostory_name}}/dc-prod.yml

```diff
@@ -11,7 +11,7 @@ services:
     logging: &logging
       driver: journald
       options:
-        tag: '{{.Name}}'
+        tag: {% raw -%}'{{.Name}}'{%- endraw %}
```

Comment:
without the `{% raw %}` cookiecutter tries to find `.Name` variable in its configuration